tools/analysis_tools/get_flops.py

```diff
@@ -29,13 +32,13 @@ def parse_args():
         'in xxx=yyy format will be merged into config file. For example, '
         "'--cfg-options model.backbone.depth=18 model.backbone.with_cp=True'")
     parser.add_argument(
-        '--shape',
+        '--input-shape',
         type=int,
         nargs='+',
         default=[256, 192],
         help='input image size')
     parser.add_argument(
-        '--input-constructor',
+        '--batch-input',
         '-c',
         type=str,
         choices=['none', 'batch'],
```

Comment:
Perhaps the argument can be converted to a boolean type using the `store_true` action? Additionally, the option '-c' may no longer be intuitive, as the argument name has been altered.

Reply:
> Perhaps the argument can be converted to a boolean type using the `store_true` action? Additionally, the option '-c' may no longer be intuitive, as the argument name has been altered.

agree! I will update this in the next commit